### ISSUE TITLE
training set stats

### ISSUE THREAD

**Issue by priamai** [ananyalahiri2003/sigma_attack#2]
In the load parser function please add a simple tag counter, this will give an indication about how unbalanced is the dataset.
We expect to see some fat tail distribution.
Cheers

**Comment by ananyalahiri2003** [comment]
For sigma/rules/windows/driver_load/ subfolder I ran the code in src/parse_files.py and saved the tag distribution - stats look as follows - 
{
    "attack.privilege_escalation": 9,
    "attack.t1543.003": 7,
    "attack.t1068": 4,
    "attack.collection": 1,
    "attack.defense_evasion": 1,
    "attack.t1599.001": 1,
    "attack.t1557.001": 1,
    "attack.persistence": 1,
    "cve.2021.21551": 1,
    "attack.t1543": 2
}

Please let me know if this is what you were after. Thanks

**Comment by priamai** [comment]
Yes this is good, let's run stats for each folder and create a summary as Markdown.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by priamai** [comment]
We could start with all the windows and linux folders, it would be nice to also auto produce some bar charts as images @ananyalahiri2003

**Comment by priamai** [comment]
We could just put the results so far on the wiki page: https://github.com/ananyalahiri2003/sigma_attack/wiki
